go.mod

```diff
@@ -21,7 +21,7 @@ require (
 	github.com/briandowns/spinner v0.0.0-20180123222039-b3ed21e9d3b2
 	github.com/buger/goterm v0.0.0-20180307092342-c9def0117b24
 	github.com/buger/jsonparser v0.0.0-20180808090653-f4dd9f5a6b44
-	github.com/chef/go-chef v0.4.1-0.20200330214506-13464279d3bb
+	github.com/chef/go-chef v0.4.1-0.20200403221625-4ab727f0c909
```

Comment:
Pull the latest master for `chef/go-chef` lib in order to take the latest changes for policyfiles endpoints.